src/frontend/shortlist/src/components/layout/profile/UserDetails.vue

```diff
@@ -306,19 +302,6 @@ main {
   border-radius: 50%;
 }
 
-.delparent {
-  background: transparent;
-  border: 1px solid #000000;
-  padding: 5px 5px;
-  color: #000000;
-  border-radius: 3px;
-  position: relative;
-  transition: all 0.3s ease-in-out;
-  cursor: pointer;
-  overflow: hidden;
-  left: 120px;
-  margin: 2px;
-}
 .delparent:before {
```

Comment:
If you are removing the "Delete Account Permanently" option from the profile, can you also remove all "delparent" styles(that are .delparent:before, .delparent:hover and .delparent:hover:before)?

Reply:
Ok, though Pooja you should keep it on your end, in the end if you got free time we could consider add this option

Reply:
I have suggested it because you removed the button and only one style class related to it. Consider removing fully or discarding these changes so that I can handle them altogether.

Reply:
Yup, lemme check

Reply:
could you check again if I removed the correct css blocks?